CAAppKitBridge/Demo/DemoController.m

```diff
@@ -27,53 +26,101 @@
 
 #import "DemoController.h"
 
+@interface DebugButtonCell : NSButtonCell
+
+@end
+
+@implementation DebugButtonCell
+- (void) _drawBorderAndBackgroundWithFrame: (NSRect)cellFrame
+                                    inView: (NSView*)controlView
+{
+  NSLog(@"kmu");
```

Comment:
```suggestion
  // ...
```

We can leave this as a semo of what can be overridden, but let's remove the message.